content/en/account_management/guide/usage-attribution-migration.md

```diff
@@ -0,0 +1,479 @@
+the month for each tag. The [Get monthly usage attribution](https://docs.datadoghq.com/api/latest/usage-metering/#get-monthly-usage-attribution)
+API provides this same data.
+
+See the below sections for differences between the two APIs and recommendations for migrating to the new API.
+
+#### Response format
+
+The v1 API response contains a link to a zip file, which contains a tsv file for each chosen tag.
+
+The v2 API response returns the usage attribution data in json format.
+
+In order to migrate to the v2 API, your processes should handle the data in json format. You can apply transformations
+as needed to the json data in order to get it into the format that best suits your needs.
+
+#### Tag breakdown
+
+In the v1 API, there is a separate tsv file for each chosen tag.
+
+In the v2 API, you can select the tag breakdown by supplying a tag configuration in `tag_breakdown_keys` as a comma
+separated list.
+
+In order to migrate to the v2 API, make requests with each tag specified on its own in `tag_breakdown_keys`.
+
+#### Tag values
+
+In the v1 API, if a resource is tagged with the same tag multiple times that will show as a pipe (`|`) separated string
+in the column of the tag.
+
+Example:
+
+```
+month   public_id       team        infra_host_usage ....
+2022-01 abc123          billing|sre 100
+...
+```
+
+In the v2 API, the value corresponding to each tag key in the tags object is an array. If a resource is tagged with the
+same tag multiple times there will be multiple items in this list.
+
+Example:
+
+```
+...
+  "tags": {
+    "team": [
+      "billing",
+      "sre"
+    ]
+  },
+...
+```
+
+In order to migrate to the v2 API, your processes should handle resources with the same tag applied multiple times.
+Tag values in the v2 response array will appear in the same order as they appear in the pipe separated string in the v1
+response, so you can join the array with pipe characters to produce the same tag values as the v1 response.
+
+#### Total usage
+
+In the v1 API, the second row of the file contains aggregated usage for all tags.
+
+In the v2 API, the `metadata.aggregates` section of the response contains aggregated usage for all tags.
+
+In order to migrate to the v2 API, retrieve total usage from the `metadata.aggregates` section.
+
+#### Usage data type
+
+In the v1 API, some usage is returned with decimal precision. Example:
+
+```
+... container_usage ...
+... 55.4            ...
+...
+```
+
+In the v2 API, usage is returned with integer precision. Example:
+`"container_usage": 55`
+
+It is not possible to convert from the integer values to the decimal values, but the integer values will be the rounded
+values of the decimal values.
+
+#### Handling of child orgs
+
+In the v1 API, the file only contains data for the tag configuration set on the parent org. This includes any child orgs
+of the parent because tag configurations are also applied to child orgs.
```

Comment:
```suggestion
In the v1 API, the file contains only data for the tag configuration set on the parent org. This includes any child orgs
of the parent, because tag configurations are also applied to child orgs.
```